classification.py

```diff
@@ -0,0 +1,38 @@
+import tensorflow as tf
+import numpy as np
+import cv2 as cv
+
+class Classification:
```

Comment:
We will have two classes for TensorFlow and OpenVINO classification - let's name correspondingly.